nebula_decoders/src/nebula_decoders_velodyne/decoders/vlp16_decoder.cpp

```diff
@@ -230,7 +261,10 @@ void Vlp16Decoder::unpack(const velodyne_msgs::msg::VelodynePacket & velodyne_pa
                 const float z_coord = distance * sin_vert_angle;       // velodyne z
                 const uint8_t intensity = current_block.data[k + 2];
 
-                double point_time_offset = timing_offsets_[block][firing * 16 + dsr];
+                auto block_timestamp = rclcpp::Time(velodyne_packet.stamp).seconds();
```

Comment:
I think this one was already defined before. Can you check?

Reply:
Fixed in https://github.com/tier4/nebula/commit/12077a81dd3969d7210baaf1a92440ee132bc096